common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -151,12 +151,28 @@ def has(self, key):
         else:
             return False
 
+    def __repr__(self):
+        return "MongoKeyValueStore{!r}<{!r}, {!r}>".format(
+            (self._data, self._parent, self._children, self._metadata),
+            self._fields,
+            self.inherited_settings
+        )
+
```

Comment:
I understood a `__repr__` to be an unique identification of this object and only this object. Is it OK that two object instances with the exact same field values would have the same `__repr__` string? I'm not implying that it's wrong - just want to know your philosophy here.